mod-data-import-converter-storage-server/src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -0,0 +1,79 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.jaxrs.model.TenantAttributes;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.tools.utils.TenantTool;
+
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Map;
+
+public class ModTenantAPI extends TenantAPI {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ModTenantAPI.class);
+  private static final String TEST_MODE = "test.mode";
+  private static final String SETUP_TEST_DATA_SQL = "templates/db_scripts/setup_test_data.sql";
+  private static final String TENANT_PLACEHOLDER = "${myuniversity}";
+  private static final String MODULE_PLACEHOLDER = "${mymodule}";
+
+  @Validate
+  @Override
+  public void postTenant(TenantAttributes entity, Map<String, String> headers, Handler<AsyncResult<Response>> handlers, Context context) {
+    super.postTenant(entity, headers, ar -> {
+      if (ar.failed()) {
+        handlers.handle(ar);
+      } else {
+        setupTestData(headers, context)
+          .setHandler(event -> handlers.handle(ar));
+      }
+    }, context);
+  }
+
+  private Future<List<String>> setupTestData(Map<String, String> headers, Context context) {
+    try {
+      if (!Boolean.TRUE.equals(Boolean.valueOf(System.getenv(TEST_MODE)))) {
+        LOGGER.info("Module is being deployed in production mode");
+        return Future.succeededFuture();
+      }
+
+      InputStream inputStream = this.getClass().getClassLoader().getResourceAsStream(SETUP_TEST_DATA_SQL);
```

Comment:
Try to simplify using FileUtils from Apache

Reply:
Tried the suggested solution locally and for some reason it didn't work. I suggest merging this PR as it is  since it is a working solution, and I will spend some time later figuring out what the problem is.